exporter/awsemfexporter/metric_translator.go

```diff
@@ -358,6 +367,24 @@ func translateCWMetricToEMF(cWMetric *cWMetrics, config *Config) (*cwlogs.Event,
 	// convert CWMetric into map format for compatible with PLE input
 	fieldMap := cWMetric.fields
 
+	for key := range fieldMap {
+		if _, ok := prometheusInstanceScrapeMetrics[key]; ok {
+			config.logger.Debug(
+				"Detected prometheus job instance metrics",
+				zap.Any("key", key),
+				zap.Any("value", fieldMap[key]),
+			)
+			// Return empty logEvent for prometheusInstanceScrapeMetrics if EnhancedContainerInsights is enabled
+			if config.EnhancedContainerInsights {
+				logEvent := cwlogs.NewEvent(
```

Comment:
Why return an empty log event? We could have other fields that customers might need in the same event.

Cant we just drop the specific fields?